Description: Unset build_tag so the installed module isn't "dev" suffixed.
Author: Neil Williams <neil@spladug.net>
Forwarded: not-needed
Last-Update: 2012-12-24
Index: kazoo/setup.cfg
===================================================================
--- kazoo.orig/setup.cfg	2012-12-21 19:25:50.649997478 -0800
+++ kazoo/setup.cfg	2012-12-23 22:43:45.557703554 -0800
@@ -1,5 +1,5 @@
 [egg_info]
-tag_build = dev
+tag_build =
 
 [nosetests]
 where=kazoo
